numpy/_core/tests/test_datetime.py

```diff
@@ -1154,26 +1154,26 @@ def test_datetime_add(self):
     def test_datetime_subtract(self):
         for dta, dtb, dtc, dtd, dte, dtnat, tda, tdb, tdc in \
                     [
-                     # One-dimensional arrays
-                     (np.array(['2012-12-21'], dtype='M8[D]'),
-                      np.array(['2012-12-24'], dtype='M8[D]'),
-                      np.array(['1940-12-24'], dtype='M8[D]'),
-                      np.array(['1940-12-24T00'], dtype='M8[h]'),
-                      np.array(['1940-12-23T13'], dtype='M8[h]'),
-                      np.array(['NaT'], dtype='M8[D]'),
-                      np.array([3], dtype='m8[D]'),
-                      np.array([11], dtype='m8[h]'),
-                      np.array([3*24 - 11], dtype='m8[h]')),
-                     # NumPy scalars
-                     (np.datetime64('2012-12-21', '[D]'),
-                      np.datetime64('2012-12-24', '[D]'),
-                      np.datetime64('1940-12-24', '[D]'),
-                      np.datetime64('1940-12-24T00', '[h]'),
-                      np.datetime64('1940-12-23T13', '[h]'),
-                      np.datetime64('NaT', '[D]'),
-                      np.timedelta64(3, '[D]'),
-                      np.timedelta64(11, '[h]'),
-                      np.timedelta64(3*24 - 11, '[h]'))]:
+                        # One-dimensional arrays
+                        (np.array(['2012-12-21'], dtype='M8[D]'),
+                         np.array(['2012-12-24'], dtype='M8[D]'),
+                         np.array(['1940-12-24'], dtype='M8[D]'),
+                         np.array(['1940-12-24T00'], dtype='M8[h]'),
+                         np.array(['1940-12-23T13'], dtype='M8[h]'),
+                         np.array(['NaT'], dtype='M8[D]'),
+                         np.array([3], dtype='m8[D]'),
+                         np.array([11], dtype='m8[h]'),
+                         np.array([3*24 - 11], dtype='m8[h]')),
+                        # NumPy scalars
+                        (np.datetime64('2012-12-21', '[D]'),
+                         np.datetime64('2012-12-24', '[D]'),
+                         np.datetime64('1940-12-24', '[D]'),
+                         np.datetime64('1940-12-24T00', '[h]'),
+                         np.datetime64('1940-12-23T13', '[h]'),
+                         np.datetime64('NaT', '[D]'),
+                         np.timedelta64(3, '[D]'),
+                         np.timedelta64(11, '[h]'),
+                         np.timedelta64(3*24 - 11, '[h]'))]:
```

Comment:
Hmm, I think here and below the contents are over indented. Might be a better way to write these, maybe put the opening `[` on the preceding line and dispense with the line continuation?